justfile

```diff
@@ -0,0 +1,10 @@
+# Automatically format all bzl files. You probably want to do this before committing.
+# It is super slow though :/.
+format:
+    #!/usr/bin/env bash
+    set -euo pipefail
```

Comment:
I got a annoyed at the format tests constantly failing so I added this as a short hand to fix things. There doesn't seem to be one super command that fixes everything unfortunately.

I could add this to husky, but it's super slow so I didn't. 😅